v2x_ros_driver/launch/v2x_ros_driver.launch.py

```diff
@@ -76,32 +85,43 @@ def generate_launch_description():
         on_exit= Shutdown()
     )
 
-    ros2_cmd = launch.substitutions.FindExecutable(name='ros2')
-
-    process_configure = launch.actions.ExecuteProcess(
-        cmd=[ros2_cmd, "lifecycle", "set", "/v2x_ros_driver", "configure"],
+    # Node lifecycle activation called only if enable_v2x_driver_lifecycle is set to true
```

Comment:
I think this comment could be moved closer to the `activate_node_group_action`, or its `IfCondition`, if you agree?